jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/ColumnBatch.java

```diff
@@ -28,7 +30,7 @@
    * This function is be called by native code during iteration and can be made as private
    * method.  We keep it as public simply to silent the linter.
    */
-  public final String getArrayInterfaceJson() {
+  private final String getArrayInterfaceJson() {
```

Comment:
There's a warning about this method not being used when it's private.  If you change it to public could you please also update the comments above?

Reply:
where is the warning? I didn't see that

Reply:
Done.